client/client.go

```diff
@@ -293,3 +295,38 @@ func (lc *LogClient) postAndParse(path string, body []byte, rsp interface{}) (*H
 	}
 	return httpData, nil
 }
+
+// AddChain performs an add-chain request, posting the provided certificate
+// chain to the CT Log hosted at LogClient.url.  The first certificate in chain
+// should be the end-entity certificate, with the second chaining to the first
+// and so on to the last, which should either be the root certificate or a
+// certificate that chains to a root certificate that is accepted by the Log.
+// Returned is:
+//   - a populated ct.SignedCertificateTimestamp, if no error is returned.
+//   - an HTTPData struct (see above).
+//   - an error, which could be a normal error, any of the error types listed in
```

Comment:
(per offline chat)  'normal error', normal seems a bit vague / subjective.

You could say: 'errors from the Go standard library'
or just 'non-Monologue errors'